registry/cache/warming.go

```diff
@@ -46,8 +46,8 @@ type backlogItem struct {
 	registry.Credentials
 }
 
-// Continuously get the images to populate the cache with, and
-// populate the cache with them.
+// Loop continuously get the images to populate the cache with,
```

Comment:
Better fix the verbs before the grammar police spot this ..

Reply:
Sorry. Fixed with `Loop continuously gets` (as I guess that's what you were referring to).